modules/message-lane/src/benchmarking.rs

```diff
@@ -274,6 +397,86 @@ benchmarks_instance! {
 			20,
 		);
 	}
+
+	// Benchmark `receive_messages_delivery_proof` extrinsic where single relayer delivers multiple messages.
+	receive_delivery_proof_for_multiple_messages_by_single_relayer {
+		// there actually should be used value of `MaxUnrewardedRelayerEntriesAtInboundLane` from the bridged
+		// chain, but we're more interested in additional weight/message than in max weight
```

Comment:
I'm confused with this comment. Do you mean that we should be using a "real" value from one of the runtimes, but it doesn't matter since we only care about each _additional_ message?

Reply:
Yes. I mean that someone probably want the weight for the worst case (when there are `MaxUnrewardedRelayerEntriesAtInboundLane` relayers rewarded in the single tx). So this comment here is to warn that this isn't necessary the value you're interested in.